src/project-builder.ts

```diff
@@ -29,33 +28,12 @@ export const projectBuilder = async function () {
     return;
   }
 
-  // @ts-ignore https://github.com/enquirer/enquirer/issues/379
-  const confirmer = new prompt.Confirm({
-    name: "mkdirartifacts",
-    message: "Do you want to clone Tableland's Validator repository?",
-  });
-
-  const mkdirArtifacts = await confirmer.run();
-
-  if (!mkdirArtifacts) {
+  // check for existing config and bail if it exists
+  if (configExists()) {
```

Comment:
Don't want to overwrite an existing config in case someone runs this on accident